Exiled.Loader/TranslationManager.cs

```diff
@@ -0,0 +1,169 @@
+// -----------------------------------------------------------------------
+// <copyright file="TranslationManager.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Loader
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Exiled.API.Extensions;
+    using Exiled.API.Features;
+    using Exiled.API.Interfaces;
+    using Exiled.Loader.Features.Configs;
+    using YamlDotNet.Core;
+    using YamlDotNet.Serialization;
+    using YamlDotNet.Serialization.NamingConventions;
+    using YamlDotNet.Serialization.NodeDeserializers;
+
+    /// <summary>
+    /// Used to handle plugin translations.
+    /// </summary>
+    public static class TranslationManager
+    {
+        /// <summary>
+        /// Gets the translation serializer.
+        /// </summary>
+        public static ISerializer Serializer { get; } = new SerializerBuilder()
+            .WithTypeInspector(inner => new CommentGatheringTypeInspector(inner))
+            .WithEmissionPhaseObjectGraphVisitor(args => new CommentsObjectGraphVisitor(args.InnerVisitor))
+            .WithNamingConvention(UnderscoredNamingConvention.Instance)
+            .IgnoreFields()
+            .Build();
+
+        /// <summary>
+        /// Gets the translation deserializer.
+        /// </summary>
+        public static IDeserializer Deserializer { get; } = new DeserializerBuilder()
+            .WithNamingConvention(UnderscoredNamingConvention.Instance)
+            .WithNodeDeserializer(inner => new ValidatingNodeDeserializer(inner), deserializer => deserializer.InsteadOf<ObjectNodeDeserializer>())
+            .IgnoreFields()
+            .IgnoreUnmatchedProperties()
+            .Build();
+
+        /// <summary>
+        /// Loads all plugin translations.
+        /// </summary>
+        /// <param name="rawTranslations">The raw translations to be loaded.</param>
+        /// <returns>Returns a dictionary of loaded translations.</returns>
+        public static SortedDictionary<string, ITranslations> Load(string rawTranslations)
+        {
+            try
+            {
+                Log.Info("Loading plugin translations...");
+
+                Dictionary<string, object> rawDeserializedTranslations = Deserializer.Deserialize<Dictionary<string, object>>(rawTranslations) ?? new Dictionary<string, object>();
+                SortedDictionary<string, ITranslations> deserializedTranslations = new SortedDictionary<string, ITranslations>(StringComparer.Ordinal);
+
```

Comment:
Shouldn't we support translations for `Exiled.Loader`?

Reply:
I'm a bit confused about what you mean by this. The highlighted section doesn't make much sense with it either. Could you elaborate on it a bit?

Reply:
I think I understand. We could add translations to the loader, but it only ever sends messages to the console and I'd rather spend time implementing the actual translation feature. As with the multiple translations, you can add it later if you want, but I don't think it would really be worth the effort to change the messages at the start of the console.

Reply:
Exactly that.
`Exiled.Loader` could print some errors/warnings about plugin/dependencies not loading correctly or outdated plugins, so in my opinion it would be useful.